src/main/java/org/candlepin/controller/ContentAccessMode.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2009 - 2024 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.controller;
+
+/**
+ * The ContentAccessMode enum specifies the supported content access modes, as well as some
+ * utility methods for resolving, matching, and converting to database-compatible values.
+ */
+public enum ContentAccessMode {
+    /** traditional entitlement mode requiring clients to consume pools to access content */
+    ENTITLEMENT,
+
+    /** simple content access (SCA) mode; clients have access to all org content by default */
+    ORG_ENVIRONMENT;
+
+    /**
+     * Fetches the default content access mode list database value
+     *
+     * @return the default content access mode list database value as a string
+     */
+    public static String defaultContentAccessModeList() {
```

Comment:
I'm not much a fan of this method living here. Its output is not a list of enums, and the default list isn't a function of this type. This is a CA utility method living on a type, which feels kinda weird when we already have the CA manager to host such things.

To some degree, the decision to hard code the default mode and list was probably a mistake. These should (have) be(en) configurable.

Reply:
Do you want to move it back? I only wanted to move the enum as the nesting when using it from outside was annoying. The method looked like it might be a good enough fit there as well, but I can move it back.

Reply:
@nikosmoum @joshmalbrecht Thoughts on this?

Reply:
The enum class should be about the building block (the mode - org_env or entitlement), while the 'mode list' is at a higher level, using the building block, but not necesserily belonging to it. So I'd say this method fits the COntentAccessManager more